pootle/apps/pootle_app/management/commands/contributors.py

```diff
@@ -87,3 +139,13 @@ def list_contributions(self, contribs, sort_by):
 
         for line in out:
             self.stdout.write(line)
+
+    def list_only_emails(self, units):
```

Comment:
Don't need an answer but wondered why this couldn't be folded into the other output format.  It seems to just be an extra exclusion and a different format.  Seems overkill and confusing to have its own method.  Just smells a bit.